viadot/sources/business_core.py

```diff
@@ -141,18 +141,16 @@ def to_df(self, if_empty: str = "skip") -> pd.DataFrame:
             APIError: When selected API view is not available.
         """
         view = self.url.split("/")[-1]
+
         if view not in ["GetCustomerData", "GetItemMaster", "GetPendingSalesOrderData"]:
             raise APIError(f"View {view} currently not available.")
-        if view in ("GetCustomerData", "GetItemMaster"):
-            data = self.get_data().get("MasterDataList")
-            df = pd.DataFrame.from_dict(data)
-            logger.info(
-                f"Data was successfully transformed into DataFrame: {len(df.columns)} columns and {len(df)} rows."
-            )
-            if df.empty:
-                self._handle_if_empty(if_empty)
-            return df
-
-        if view == "GetPendingSalesOrderData":
-            # todo waiting for schema
-            raise APIError(f"View {view} currently not available.")
+
+        data = self.get_data().get("MasterDataList")
+        df = pd.DataFrame.from_dict(data)
+        logger.info(
+            f"Data was successfully transformed into DataFrame: {len(df.columns)} columns and {len(df)} rows."
+        )
+        if df.empty:
+            self._handle_if_empty(if_empty)
```

Comment:
Please add information what option the user has in `if_empty`. So far only `skip` is known.

Reply:
By the way, such of variables should have prefix `has`, `is` or suffix `able` or without if an object, not `if`.

Examples:
```python

# good
user.empty() 
user.is_empty()
user.active()
user.is_active()

# less good, but good
user.has_profile() # <- has some profile data
user.has_activation() # <- meens is not active, but has the ability.
user.has_data() # <- too abstract but ok

# good, if need a bool pre-check 
user.activatable()
user.writable()


# other properties bool-methods
user.has_name()
user.is_named()


# not so good
user.if_empty()
``

Reply:
Thanks guys! I applied changes from @Rafalz13 and @Diego-H-S but @eSlider suggestion will require changes in multiple viadot files - maybe it's good for future improvements and can be added as an issue?